addons/website/static/src/components/dialog/seo.js

```diff
@@ -174,6 +285,32 @@ class Keyword extends Component {
         return this.website.pageDocument.body.textContent;
     }
 
+    get mentionedIn() {
+        return [
+            this.usedInH1 && "H1",
+            this.usedInH2 && "H2",
+            this.usedInTitle && "Title",
+            this.usedInDescription && "Description",
+            this.usedInContent && "Body",
+        ]
+            .filter(Boolean)
+            .join(", ");
+    }
+
+    get notMentionedIn() {
+        const res = [
+            !this.usedInH1 && "H1",
+            !this.usedInH2 && "H2",
+            !this.usedInTitle && "Title",
+            !this.usedInDescription && "Description",
+            !this.usedInContent && "Body",
+        ].filter(Boolean);
+        if (res.length === 5) {
+            return _t("Not in the page");
+        }
+        return res.join(", ");
```

Comment:
`_t()` should be applied to the final result (of `join()`) no ?

Reply:
Yep, forgot it. :+1: